src/main/xcontainers/XSourcePanel.js

```diff
@@ -16,18 +19,34 @@ class SourcePanel extends Component {
         style={{ cursor: 'pointer' }}
         >
         <h1>Source</h1>
+        <RequestBar createBodyFromSource={this.props.createBodyFromSource}/>
         <BodyItemsContainer
           collection='CLONED_ITEMS'
-          items={this.props.clonedItems}
+          // items={this.props.clonedItems}
```

Comment:
Just delete things like this, please. No need to have comments all over - right?